services/gateway/apis/http-version1/methods/votesSent.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
 		secpubkey: { optional: true, type: 'string', min: 64, max: 64 },
 		limit: { optional: true, min: 1, max: 100, type: 'number' },
 		offset: { optional: true, min: 0, type: 'number' },
+		onefrom: ['address', 'username', 'publickey', 'secpubkey'],
```

Comment:
This can be defined separately at the upper level property. Ideally as array of arrays that define one of the properties that must be set.

Reply:
Done!